File: 2026-09/src/day_11/prms_181865/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

int solution(string binomial) {
    stringstream ss(binomial);
    int a, b;
    string op;
    ss >> a >> op >> b;

    if (op == "+") return a + b;
    if (op == "-") return a - b;
    return a * b;
}

File: 2026-09/src/day_11/prms_181865/Solution.java
package day_11.prms_181865;

import java.util.*;

class Solution {
    public int solution(String binomial) {
        StringTokenizer st = new StringTokenizer(binomial);
        int a = Integer.parseInt(st.nextToken());
        String op = st.nextToken();
        int b = Integer.parseInt(st.nextToken());

        if (op.equals("+")) return a + b;
        if (op.equals("-")) return a - b;
        return a * b;
    }
}
